app/models/user.rb

```diff
@@ -1,13 +1,13 @@
 class User < ApplicationRecord
-  # Associations
+  devise :database_authenticatable, :registerable, :confirmable, :recoverable, :rememberable, :validatable
   has_many :posts, foreign_key: 'author_id'
   has_many :comments
   has_many :likes
 
   # Validations
   validates :name, presence: true
   validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-
+  devise :confirmable
   # Methods
```

Comment:
- [ ] You already added the `confirmable` module in line 2 there is no need to repeat it again in line 10. please remove line 10 to achieve the `DRY` principle as the previous reviewer suggested.